### ISSUE TITLE
No buttons to download videos on the new layout

### ISSUE THREAD

**Issue by furyutei** [furyutei/twMediaDownloader#21]
Please report on the following:

1. Your browser and extension version (e.g. Google Chrome Version 76.0.3809.100 (Official Build) (64-bit), extension version 0.1.2.301)
1. Your option values (attach a snapshot of the options popup)
1. Are there download buttons when displaying tweets with GIF attached in the new layout? (For example, [these tweets on search timeline](https://twitter.com/search?q=card_name%3Aanimated_gif&src=typed_query&f=live))
1. If you opened [this Tweet](https://twitter.com/furyutei/status/817020036682420225) in the new layout, is the download button displayed?
1. If you opened [this Tweet](https://twitter.com/furyutei/status/817020036682420225) in the old layout (enable the "Allow in incognito" of this extension(chrome://extensions/?id=cfgbdcfdnajmlllbdgincbpdjmcldieb) and display in an incognito window, or use the "GoodTwitter" extension), is the download button displayed?

**Comment by furyutei** [comment]
https://twitter.com/btsfirefly/status/1160706260200824832
Umm...perhaps there is a difference in the displayed page structure depending on the user.
Try the following:
1. Open [this Tweet](https://twitter.com/furyutei/status/817020036682420225) in the new layout.
1. Open the Developer tools(F12 or Ctrl+Shift+I) and click "Console" tab.
1. Evaluate following expressions(copy, paste and press Enter on console) and report results(attach a snapshot of Console). 
    ```Javascript
    console.log(document.querySelector( 'div[data-testid="primaryColumn"] article[role="article"] div[data-testid="previewInterstitial"] div[style*="background-image"]' ));
    console.log(document.querySelector( 'div[data-testid="previewInterstitial"]' ));
    console.log(document.querySelector( 'div[style*="/ext_tw_video_thumb/"]' ));
    ```
    e.g. ![0 1 2 301-VideoTweet-console-log](https://user-images.githubusercontent.com/7148490/62842089-3b9c9f00-bcea-11e9-8c39-fb389539ae5e.png)

**Comment by furyutei** [comment]
I found that the download button is not displayed when video autoplay is enabled,

> More > Settings and privacy > General > Data Usage > Video autoplay
> Check "On cellular or Wi-Fi"

[This issue has been fixed in version 0.1.2.401.](https://github.com/furyutei/twMediaDownloader/commit/7ba8fd2edfd2668c40ab70fa06f82b14fc3fa1db)

動画の自動再生を有効にしていると、ダウンロードボタンが表示されないことがわかりました。

> もっと見る > 設定とプライシー > 一般 > データ通信量 > 動画の自動再生
> 「モバイルデータ通信とWi-Fi接続時」にチェック

[この問題は version 0.1.2.401 にて修正しました。](https://github.com/furyutei/twMediaDownloader/commit/7ba8fd2edfd2668c40ab70fa06f82b14fc3fa1db)
